src/libtriton/format/abstractBinary.cpp

```diff
@@ -116,6 +122,13 @@ namespace triton {
     }
 
 
+    triton::format::pe::PE* AbstractBinary::getPE(void) {
```

Comment:
To get the same convention style than all others classes, the class `PE` should be `Pe` and the method name `getPe(void)`